app/models/organization.js

```diff
@@ -0,0 +1,29 @@
+import mongoose from 'mongoose';
+
+var Schema = mongoose.Schema;
+
+var OrgSchema = new Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  email: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    required: true
+  },
+  founded: {
+    type: Date,
+    default: Date.now
```

Comment:
Can we remove this default `founded` date?

I agree with the decision to make the field not required, but I can't think of a case where the founding date should be chosen as the current date since users of this platform are always registering after founding.